test/mocks/cams_mock.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Gringotts.Gateways.CamsMock do
```

Comment:
@gopalshimpi the indentation is totally out of place in this whole use 2 spaces and verify once in github also.

>> Tip: when you are creating a new file in the starting change its setting to use spaces for indentation and that too 2 spaces.